Index: src/L01WorkingWithAbstraction/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package L01WorkingWithAbstraction;\r\n\r\npublic class Main {\r\n    public static void main(String[] args) {\r\n        System.out.println();\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/L01WorkingWithAbstraction/Main.java b/src/L01WorkingWithAbstraction/Main.java
--- a/src/L01WorkingWithAbstraction/Main.java	
+++ b/src/L01WorkingWithAbstraction/Main.java	
@@ -2,6 +2,5 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println();
     }
 }
